sources/TileDB.CSharp/Marshalling/SafeHandles/ContextHandle.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Runtime.InteropServices;
+using TileDB.CSharp;
+using TileDB.Interop;
+
+namespace TileDB.CSharp.Marshalling.SafeHandles
+{
+    internal unsafe sealed class ContextHandle : SafeHandle
+    {
+        public ContextHandle() : base(IntPtr.Zero, true) { }
+
+        public ContextHandle(IntPtr handle, bool ownsHandle) : base(IntPtr.Zero, ownsHandle) { SetHandle(handle); }
+
+        public static ContextHandle CreateUnowned(tiledb_ctx_t* schema) => new((IntPtr)schema, ownsHandle: false);
```

Comment:
It's a variable name so just a nit, but I think `schema` should be `context` here. Similar typo for this variable on other safehandle classes.